client/src/app/styles/_mixins.scss

```diff
@@ -0,0 +1,5 @@
+@mixin medium-screens(){
+  @media only screen and (min-width: 800px) {
+    @content;
+  }
+}
```

Comment:
More flexible solution will be to define breakpoints as const in some sass file maybe _variables.scss 

```
 $small: 0px,
 $medium: 640px,
 $large: 1024px,
 $xlarge: 1200px,
 $xxlarge: 1440px,
```

and then write mixin
```
@mixin breakpoint($width) { 
   @media only screen and (min-width: $width) {
      @content;
    }
}
```

and then we will simply use 
```
@include breakpoint($medium) {
 ...some styles here
}
```